unittest/gunit/opt_range-t.cc

```diff
@@ -351,7 +351,9 @@ void OptRangeTest::check_use_count(SEL_TREE *tree)
   {
     SEL_ARG *cur_range= tree->keys[i];
     if (cur_range != NULL)
+    {
```

Comment:
What warning was fixed here?

Reply:
The macro below this contains an if-else without brackets, so it generated a warning about ambiguity.